### DISCUSSION TITLE
Unable to access codespaces via VSCode from a specific organization, other org codespaces still show up.

### DISCUSSION THREAD

**Discussion by mikebronner** [community/community#58277] [Codespaces]
### Select Topic Area

Bug

### Body

About 15 minutes ago i got an error while working in a codespace in VSCode (most recent build, not Insiders) that the codespace was deleted or unavailable. Looking from the github web ui, i can still see the codespace listed, but when browsing codespaces in VSCode, one entire org's codespaces are missing, but codespaces from another org are visible.

When I try to open the codespace from the web in VSCode I get the following error:
```
Connect to Codespace: Failed to connect. Requested codespace not found.
```

VSCode Insiders seems to have the same issue.

Opening the codespace in the web view does seem to work.

https://status.github.com also shows no incident.

**Comment by 275RR-Public** [comment]
Exact issue for me as well.
Can open the Codespace in the browser but not on VS Code desktop app.

**Comment by matthieu-foucault** [comment]
Tried to create a codespace from the extension as a workaround. It did create the codespace, but failed to retrieve it

![image](https://github.com/community/community/assets/2684022/e295fc0d-2249-4391-9606-f0e77f2787a8)

**Comment by tim-wingspan** [comment]
<img width="776" alt="Screenshot 2023-06-16 at 2 19 42 PM" src="https://github.com/community/community/assets/129194734/0e41c957-a141-4853-9cdd-4a9251b184b4">
<img width="509" alt="Screenshot 2023-06-16 at 2 19 58 PM" src="https://github.com/community/community/assets/129194734/bc1ade8f-97b6-4c21-a352-83021172be29">
<img width="510" alt="Screenshot 2023-06-16 at 2 22 05 PM" src="https://github.com/community/community/assets/129194734/dd0c154b-06b1-4b98-9bad-bb3799fbef9d">

**Comment by 77AIds** [comment]
Same here! All worked fine a couple of hours ago and now I just can't open the codespaces via VSCode app. Still available on the web

**Comment by ffac789** [comment]
Same issue here. Attempted a few workarounds which led to the same error. As with others, web version is still available.

**Comment by LuKaarGit** [comment]
same problem, tried updating everything and re adding codespace, same error, but yes web version does work for anyone who wants to get stuff done in the mean time lol

**Comment by WalidOumouzoune** [comment]
same problem here

**Comment by exiao** [comment]
Getting this as well

**Comment by tim-wingspan** [comment]
Resolved for me 👍

**Reply by ffac789** [reply]
Working for me now as well.

**Comment by doniyordjon** [comment]
same problem

**Comment by vediyd-calab** [comment]
same problem here
